Website: add documentation for `tag` and `titleTag` arguments (#2365)
hashicorp/design-system: website/docs/utilities/dialog-primitive/partials/code/how-to-use.md

  </:footer>
</Hds::DialogPrimitive::Wrapper>
```

### Header titleTag

> The `titleTag` value changes the HTML element wrapping the tagline and "title" block in the Dialog Header. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if the `DialogPrimitive` is used as a Split Window, the value should be `"h2"`. 

Reviewer: ```suggestion
The `@titleTag` attribute changes the HTML element that wraps the `Dialog::Header` tagline and "title" content. When organizing the content on a webpage, the heading levels should reflect the structure of the page. For example, if the `DialogPrimitive` is used as a Split Window, the value should be `"h2"`. 
```

```handlebars
<div class="doc-dialog-primitive-grid-layout">
  <div class="doc-dialog-primitive-flex-layout">
    <Hds::Text::Display @tag="h1" @size="500">Page with Split Window</Hds::Text::Display>
    <div class="doc-dialog-primitive-content-placeholder">
     <Hds::Text::Body>Main page content </Hds::Text::Body>

>     </div>

Reviewer 2: This should use a `Doc::Placeholder` component. Then you wouldn't have to add custom HTML.
